src/traversals/parallel_bfs.jl

```diff
@@ -15,7 +15,7 @@ import Base: push!, shift!, isempty, getindex
 
 export bfs_tree, LevelSynchronousBFS
 
-mutable struct LevelSynchronousBFS <: AbstractGraphVisitAlgorithm end
+struct LevelSynchronousBFS end
```

Comment:
I guess the main thing is that a single BFS is fairly fast and won't benefit much from parallelism, but the gains come when you can do multiple BFS in parallel.